### PR TITLE
FIX some small issues

### PR DESCRIPTION
**Pull request by fgalan**
LGTM from @kzangeli and @raconte needed

### CODE REVIEW

File: test/functionalTest/harnessFunctions.sh [hunk 1]
```diff
@@ -145,7 +145,7 @@ function localBrokerStart()
   traceLevels=$2
   ipVersion=$3
 
-  POOL_SIZE=1
+  POOL_SIZE=10
```
Comment (kzangeli) [inline test/functionalTest/harnessFunctions.sh]: I would prefer to do this a little better:

    POOL_SIZE=${POOL_SIZE:-10}

Meaning, if POOL_SIZE already exists, use that value, if not, set it to 10.
Can't say I like the name of the variable. CB_DB_POOL_SIZE would be a lot better ...

Comment (fgalan) [inline test/functionalTest/harnessFunctions.sh]: Fixed in a57ca79

**Comment by raconte** [comment]
LGTM

**Comment by kzangeli** [comment]
LGTM

**Pull request merged**